shared/js/textChatView.jsx

```diff
@@ -79,6 +80,23 @@ loop.shared.views.chat = (function(mozL10n) {
         );
       }
 
+      if (this.props.contentType === CHAT_CONTENT_TYPES.NOTIFICATION) {
+
+        var thumbnail = this.props.useDesktopPaths ?
+            "shared/img/leave_notification.svg" :
+            "shared/img/leave_notification.svg";
```

Comment:
No need for this since these are the same now. In fact, I've just raised a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1247190) to drop the use of useDesktopPaths.